### ISSUE TITLE
No Response to MyQ accessory in Homekit

### ISSUE THREAD

**Issue by robertz27** [iRayanKhan/homebridge-chamberlain#112]
# Explain the issue occuring:
I loaded the HOOBs version of homebridge on my raspberry pi as my previous version on raspberian stopped working.  When loading the plugin for MyQ, it is now asking for a name, in addition to email/password, when in the previous plugin all I needed was my email and password. I have two garage door sensors which are labeled "Right Garage Door" and "Left Garage Door".  The older version automatically loaded these.  Now I cannot get it to load properly.

# Info
iOS Version: HOOBS 3.2.6
Plugin Version:  Chamberlain
1.6.1 
HomeBridge Version:
Node/Npm Version: 

# Log Output:
(If applicable)

**Comment by iRayanKhan** [comment]
Do you have your deviceID’s by chance?

**Comment by robertz27** [comment]
They are:

Right Garage Door
Left Garage Door

**Comment by robertz27** [comment]
I found the solution by finding the instructions below.  I was confused at first by the device ID (which I thought was the device name).  I had to open the log to find the error statement which provided the device IDs.  Worked when I manually added the code into the config.json file under the Advanced link in the plugin window.

The instructions as follows:

If you have multiple garage doors, the plugin will throw an error and list the controllable device IDs. Use those IDs to create individual accessories. Be sure to uniquely name the door via the "name" field, otherwise you'll get a UUID error in the console (Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory).

{
  "accessory": "Chamberlain",
  "name": "Main Garage Door",
  "username": "your mychamberlain.com email",
  "password": "your mychamberlain.com password",
  "deviceId": "xxx"
},
{
  "accessory": "Chamberlain",
  "name": "Side Garage Door",
  "username": "your mychamberlain.com email",
  "password": "your mychamberlain.com password",
  "deviceId": "xxx"
},
...
